timemachine/fe/single_topology.py

```diff
@@ -980,21 +968,20 @@ def interpolate_harmonic_angle_params(src_params, dst_params, lamb, k_min, lambd
     src_k, src_phase, _ = src_params
     dst_k, dst_phase, _ = dst_params
 
-    k = interpolate_harmonic_force_constant(src_k, dst_k, lamb, k_min, lambda_min, lambda_max)
+    log_linear_fn = partial(interpolate.log_linear_interpolation, min_value=k_min)
+    k = padded_interpolation(log_linear_fn, src_k, dst_k, lamb, lambda_min, lambda_max)
 
-    phase = interpolate.linear_interpolation(
-        src_phase,
-        src_phase + cyclic_difference(src_phase, dst_phase, period=2 * np.pi),
-        lamb,
-    )
+    src_x = src_phase
+    dst_x = src_phase + cyclic_difference(src_phase, dst_phase, period=2 * np.pi)
+    x = padded_interpolation(interpolate.linear_interpolation, src_x, dst_x, lamb, lambda_min, lambda_max)
```

Comment:
nit: seems more descriptive to keep this named `phase`, unless there's a convention I'm unaware of